specification/sections/04-hooks.md

```diff
@@ -87,7 +87,7 @@ EvaluationContext | void before(HookContext, HookHints);
 
 #### Requirement 4.3.4
 
-> When `before` hooks have finished executing, any resulting `evaluation context` **MUST** be merged with the invocation `evaluation context` with the invocation `evaluation context` taking precedence in the case of any conflicts.
+> When `before` hooks have finished executing, any resulting `evaluation context` **MUST** be merged with the other `evaluation context`s with the following order: invocation (highest precedence), before-hook, client, api (lowest precedence).
```

Comment:
```suggestion
> When `before` hooks have finished executing, any resulting `evaluation context` **MUST** be merged with the existing `evaluation context` in the following order: invocation (highest precedence), before-hook, client, api (lowest precedence).
```

Reply:
Is this point a bit redundant with the presence of [3.2.2](https://github.com/open-feature/spec/blob/main/specification/sections/03-evaluation-context.md#requirement-322)? Can we / should we unify these somehow?

Do we also need to mention provider hooks?